src/net/sourceforge/kolmafia/persistence/AdventureQueueDatabase.java

```diff
@@ -187,22 +188,23 @@ public static RollingLinkedList<String> getZoneNoncombatQueue(String adv) {
   }
 
   public static void serialize() {
-    File file =
-        new File(KoLConstants.DATA_LOCATION, KoLCharacter.baseUserName() + "_" + "queue.ser");
-
-    try {
-      FileOutputStream fileOut = new FileOutputStream(file);
-      ObjectOutputStream out = new ObjectOutputStream(fileOut);
-
-      // make a collection with combat queue first
-      List<TreeMap<String, RollingLinkedList<String>>> queues =
-          new ArrayList<TreeMap<String, RollingLinkedList<String>>>();
-      queues.add(COMBAT_QUEUE);
-      queues.add(NONCOMBAT_QUEUE);
-      out.writeObject(queues);
-      out.close();
-    } catch (IOException e) {
-      e.printStackTrace();
+    if (allowSerializationWrite) {
```

Comment:
would probably prefer to see this as:
```
if (!allowSerializationWrite) return;
```

to make it obvious the method does nothing else

Reply:
Ok